### PR TITLE
Initial version run

### PR DESCRIPTION
**Pull request by eledwinn**
## Summary
- styling home
- scss webpack compiller
- webpack output to build folder

## Screenshot
image

## Trello Card
https://trello.com/c/i8iy8kEE/165-revisar-que-funcione-bien-el-bootstrap-de-angularjs

### CODE REVIEW

File: src/components/centered/centered.pug [hunk 1]
```diff
@@ -1,2 +1,16 @@
-span
-  | difjifafiasfoisoifaiofasoifioasiofasdiofioafajiodfioaioç
+.page
+  img.logo(ng-src='{{$ctrl.logo}}' alt='logo')
+  h1.title
+    | {{ $ctrl.title }}
+.example
+  p
```
Comment (FrankIglesias) [inline src/components/centered/centered.pug]: this line can be deleted

File: src/components/centered/centered.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.page {
+  align-items: center;
+  background-color: $wolox-blue;
+  background-image: url('./../assets/wolox_bg.png');
```
Comment (FrankIglesias) [inline src/components/centered/centered.scss]: maybe this assets has been deleted?

File: src/components/centered/centered.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+.logo {
+  max-width: 350px;
+  margin-bottom: 50px;
+}
+
+.title {
+  color: $white;
+  font-size: 30px;
+  font-weight: 500;
+  font-family: Montserrat, sans-serif;
```
Comment (FrankIglesias) [inline src/components/centered/centered.scss]: This font family should be attached to the body

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/components/centered/centered.scss (hunk 2 above)
Comment (eledwinn) [reply src/components/centered/centered.scss]: no, the asset still exists as a background.

File: src/components/centered/centered.scss (hunk 3 above)
Comment (eledwinn) [reply src/components/centered/centered.scss]: I want only the title to have that font. The rest has sans-serif.

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**